mapping-for-navigation/_posts/0000-01-04-adding-turn-restrictions.md

```diff
@@ -42,23 +42,29 @@ When we click on the green circle, the area is automatically opened in JOSM and
 
 The new version of iD, the web editor for OpenStreetMap, makes it even simpler to add turn restrictions to OpenStreetMap.
 
+### Turn restriction visual editor in iD:
+
+* iD has visual turn-restriction editor which reduces the effort of adding a turn-restriction with few clicks.
+
+| | |
+| ------- | ------- |
+| ![screen shot 2018-02-12 at 1 51 21 pm](https://user-images.githubusercontent.com/17470597/36113755-e0d99682-0ffb-11e8-8e7e-ec95fea3731c.png) | **Red outlined way** - way which gets turn-restriction a from way    **Arrows on top of surrounding way** - where all the restrictions are possible.|
+| ![tr](https://user-images.githubusercontent.com/17470597/36114289-9b125cb8-0ffd-11e8-8fde-7c3dc6b8567f.gif) |- From the highlighted way - Every selection and color change to red arrow implies a turn-restriction. |
+
 ### Adding Absolute Turn Restriction:
 ![epl4xsg](https://cloud.githubusercontent.com/assets/17887418/17285386/fe7453f6-57df-11e6-9d1d-e65ba060bc0b.gif)
 
 ### Adding Conditional Turn Restriction:
```

Comment:
I know this is not a part of this review but can we make it a simple table. Right now it looks really distracting. Let's keep the images consistent.

Reply:
Will do it thanks 👍

Reply:
Done Jothi 👍

Reply:
⚡️